test/parallel/test-vm-not-strict.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable strict, no-var, no-delete-var */
+// Related to bug report: https://github.com/nodejs/node/issues/43129
+require('../common');
+var assert = require('assert');
+var vm = require('vm');
+
+var data = [];
+var a = 'direct';
+delete a;
+data.push(a);
+
+var item2 = vm.runInThisContext(`
```

Comment:
Strange it used to pass locally when I manually ran `node file.js`. But I may have changed something when I lint it making it red in CI 🤔 I'll re-run it locally and fix it.